www/content/migration-guide-hotwire-turbo.md

```diff
@@ -50,13 +50,13 @@ document.body.addEventListener('htmx:configRequest', (event) => {
 ```
 
 * Or, use an [hx-trigger](@/attributes/hx-trigger.md) condition: `hx-trigger="submit[action(target)]"`
-  * Does not currently resolve async calls such as `fetch`. See: https://github.com/bigskysoftware/htmx/issues/912
```

Comment:
I think the "async" to "await" here is not correct. Yes, `await` is the statement but they're "asynchronous functions".